src/ng/http.js

```diff
@@ -71,14 +71,16 @@ function headersGetter(headers) {
   var headersObj = isObject(headers) ? headers : undefined;
 
   return function(name) {
+    var existingHeaderName, lowercaseName = lowercase(name);
```

Comment:
@Narretz please have a look at the existing in methods in $http - as far as I can see there is not consistent style for declaring vars but many methods declare multiple vars using one `var` keyword ([example](https://github.com/angular/angular.js/blob/d729fcf030be1d3ef37196d36ea3bf3249ee3318/src/ng/http.js#L855-L857)). One again, I'm all for consistency and clear code standards, but given lack of consistency in the current code base, I don't think style discussion should be happening to in this PR.

My point is: I'm all for common coding standards and consistent coding patterns and we can raise it on the team meeting. But given that this PR doesn't introduce different coding style, as compared to the existing code in this file, I don't think that this PR is the best place to discuss personal coding preferences... At the end of the day I don't care either way as long we all on the same page.